gsPreCICEUtils.h

```diff
@@ -313,10 +339,10 @@ typename gsBasis<T>::Ptr knotMatrixToBasis(const gsMatrix<T> & knots)
         std::copy_if(knots.row(d).begin(),knots.row(d).end(),
                      std::back_inserter(tmp),
                      [](T a){return !math::isnan(a);});
-        KVs[d] = gsKnotVector<T>(tmp);/////// MEMLEAK
-        // KVs[d] = std::move(gsKnotVector<T>(tmp));
+        // KVs[d] = gsKnotVector<T>(tmp);/////// MEMLEAK
+        KVs[d] = std::move(gsKnotVector<T>(tmp));
```

Comment:
I think you can use `give`, [see](https://github.com/gismo/gismo/blob/6ec7368a5e36c726855c5b96fdf50f03213d5696/src/gsCore/gsMemory.h#L254)